### PR TITLE
clarify how to use dogwrap with non-us1 sites

### PR DESCRIPTION
**Pull request by irabinovitch**
### What does this PR do?

clarify how to use dogwrap with non-us1 sites. 

### Motivation
Someone asked me how to do this and it seemed reasonable to update the docs.

### Additional Notes
note dogwrap doesn't appear to support ddgov or ap1 sites via CLI arguments. (see https://github.com/DataDog/datadogpy/issues/771)

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/developers/guide/dogwrap.md [hunk 1]
```diff
@@ -22,19 +22,31 @@ To install from source:
...
+{{< /site-region >}}
+
 {{< site-region region="eu" >}}
 ```bash
 dogwrap -n <EVENT_TITLE> -k <DATADOG_API_KEY> -s eu "<COMMAND>"
 ```
 {{< /site-region >}}
 
-**Note**: The `dogwrap` command sends data to the US Datadog site by default. If you need to send data to the EU site, you must include the `-s eu` option.
+**Note**: The `dogwrap` command sends data to the US Datadog site by default. If you need to send data to another site, you must include the `-s` option specifying a target site such as `eu`, `us3`, `us5`, etc.
```
Comment (rtrieu) [inline content/en/developers/guide/dogwrap.md:49]:
```suggestion
**Note**: The `dogwrap` command sends data to the US Datadog site by default. If you need to send data to another site, you must include the `-s` option specifying a target site, such as `eu`, `us3`, `us5`, etc.
```

**Review by rtrieu — COMMENTED** [review]
Non-critical suggestion, but otherwise looks good.

**Comment by irabinovitch** [comment]
I dont think the failures are from my change.

**Review by hestonhoffman — APPROVED** [review]

**Comment by hestonhoffman** [comment]
Ah, failures are due to the branch naming scheme. Our build requires a `/` in the branch name to run. This is good to merge though 👍

**Pull request merged**